addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
from slack:
> see https://github.com/alphasights/ember-scrollable/compare/master...alexander-alvarez:testing-wait?expand=1
the first commit with just wait fails tests

From my investigation it looks like we don't even need the wait helper